torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -186,7 +186,7 @@ void CudaPrinter::visit(const For* v) {
 }
 
 void CudaPrinter::visit(const Cast* v) {
-  os() << cudaDtypeCppString(v->dtype());
+  os() << "(" << cudaDtypeCppString(v->dtype()) << ")";
```

Comment:
Ouch, how did it work? :)

Reply:
I don't know when it became a thing but apparently you can cast by saying `float(x)`, like it's a constructor.  But that sorta doesn't parse if you do `unsigned char(x)`.  Clearly we need whitespace in identifiers! (I hope this reddit post is a joke: https://www.reddit.com/r/ProgrammerHumor/comments/4on81i/til_c_allows_u200b_zero_width_space_in_identifiers/ )